docs/machineregistration-reference.md

```diff
@@ -104,22 +107,51 @@ The only required value for a successful installation is the `device` key as we
   ```
 </details>
 
+<details>
+<summary>Example device-selector based on device name</summary>
+  ```yaml showLineNumbers
+  device-selector:
+  - key: Name
+    operator: In
+    values:
+    - /dev/sda
+    - /dev/vda
+    - /dev/nvme0
+  ```
+</details>
+
+<details>
+<summary>Example device-selector based on device size</summary>
+  ```yaml showLineNumbers
+  device-selector:
```

Comment:
which device will be selected if multiple disks match the size requirements (e.g. in Longhorn nodes)

Reply:
We're introducing the `device-selector` concept to make device selection 100% deterministic. "first by chance" is contrary to this.
Erroring out is also not acceptable (because of remote installations) 😉

Reply:
I guess the only way of making it 100% deterministic without having to include cumbersome stuff (such as selecting by HW serial numbers) is by defining a selector criteria that results into a unique selection. If drives are that similar they can't be discriminated by the selector I'd say we have to assume they are also exchangeable (doesn't matter where we install as they are equivalent).